lib/processors/lib/jaccard.py

```diff
@@ -0,0 +1,133 @@
+    sa75= ['1B','2A','3B','4A','5B','6B','7A','8A','9B','10A','11A','12A','13A','14A','15A','16A','17A','18B','19A','20A','21B','22B','23A','24B','25B','26A','27A','28B','29B','30A','31B','32B','33A','34A','35A','36B','37A','38A','39B','40B','41B','42A','44B','45B','46A','47A','48B','49B','50A','52A']
+    sa76= ['1B','2A','3A','4A','5A','6B','7B','8A','9B','10A','11B','12B','13B','14A','15B','16B','17A','18B','19A','20B','21B','22A','23B','24B','25B','26A','27B','28B','29A','30B','31B','32B','33A','34A','35A','36B','37A','38A','39A','40B','41B','42B','44B','45B','46A','47A','48A','49B','50B','52A']
+    sa77= ['1B','2A','3B','4A','5B','6A','7B','8A','9B','10B','11A','12B','13A','14A','15A','16B','17A','18A','19B','20B','21B','22A','23B','24B','25B','26A','27B','28B','29B','30B','31B','32B','33A','34A','35A','36A','37A','38B','39B','40A','41B','42B','44A','45B','46A','47A','48B','49A','50B','52B']
+    sa78= ['1B','2B','3A','4B','5B','6B','7B','8A','9B','10B','11A','12B','13A','14A','15B','16A','17A','18A','19B','20A','21B','22A','23A','24A','25B','26B','27B','28B','29B','30A','31B','32B','33A','34A','35A','36A','37A','38A','39A','40B','41B','42B','44B','45B','46A','47A','48A','49B','50B','52B']
+    sa79= ['1B','2A','3B','4A','5A','6A','7A','8B','9B','10A','11A','12A','13A','14A','15A','16A','17A','18A','19A','20A','21B','22A','23B','24B','25B','26A','27B','28B','29A','30A','31A','32B','33A','34B','35A','36B','37A','38A','39A','40B','41B','42A','44A','45B','46B','47A','48B','49B','50A','52A']
+    sa80= ['1A','2A','3A','4A','5A','6A','7A','8A','9B','10A','11A','12B','13A','14B','15A','16A','17A','18A','19A','20B','21A','22A','23B','24B','25B','26A','27B','28A','29B','30A','31B','32B','33A','34B','35A','36B','37A','38A','39B','40B','41B','42B','44B','45A','46B','47B','48A','49A','50A','52B']
+    sa81= ['1A','2B','3A','4B','5A','6A','7A','8B','9A','10B','11A','12A','13B','14B','15A','16A','17A','18A','19B','20B','21A','22A','23A','24B','25B','26A','27A','28B','29A','30B','31B','32B','33A','34B','35A','36A','37B','38B','39B','40B','41B','42B','44B','45B','46A','47B','48A','49A','50A','52A']
+    stored_sets = [sa1, sa2, sa3, sa4, sa5, sa6, sa7, sa8, sa9, sa10, sa11, sa12, sa13, sa14, sa15, sa16, sa17, sa18, sa19, sa20, sa21, sa22, sa23, sa24, sa25, sa26, sa27, sa28, sa29, sa30, sa31, sa32, sa33, sa34, sa35, sa36, sa37, sa38, sa39, sa40, sa41, sa42, sa43, sa44, sa45, sa46, sa47, sa48, sa49, sa50, sa51, sa52, sa53, sa54, sa55, sa56, sa57, sa58, sa59, sa60, sa61, sa62, sa63, sa64, sa65, sa66, sa67, sa68, sa69, sa70, sa71, sa72, sa73, sa74, sa75, sa76, sa77, sa78, sa79, sa80, sa81]
+
+    a = set(current_answers)
+    current_max1 = 0
+    current_max2 = 0
+    current_max3 = 0
+
+    index1 = 0
+    index2 = 0
+    index3 = 0
+
+    for i in range(0,len(stored_sets)):
```

Comment:
@mynameisfiber 

With the current dataset, this is exactly correct.

The rest just deals with having missing answers, which could be possible if we decide to ask more questions. People are less likely to fill out a huge survey, so I figured I'd support having smaller answer sets from a bigger question pool, if that makes sense.

Thoughts?